addons/base/utils.py

```diff
@@ -1,8 +1,14 @@
 import markupsafe
 from os.path import basename
+from website.settings import MFR_SERVER_URL
 
 from website import settings
 
+def get_mfr_url(target, provider_name):
```

Comment:
If target has an osfstorage_region and provider is 'osfstorage', use the mfr_url on the region. Otherwise use MFR_SERVER_URL in settings.